### PR TITLE
Upgrade styles to CSF format

### PR DESCRIPTION
**Pull request by paul-request**
# Checklist:

- [x] The commit messages follow the [convention for this project](./blob/master/docs/CONTRIBUTING.md#conventional-commits)
- [x] I have provided documentation in the notes section of the story

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -31,4 +31,8 @@
   .sbdocs.sbdocs-a, .sbdocs.sbdocs-p {
     font-size: 14px;
   }
+
```
Comment (paul-request) [inline .storybook/preview-head.html:34]: There is a bug where an empty doc block is displayed when using the component parameters to hide the code preview, so this just prevents it from displaying for now

Comment (owensgit) [reply .storybook/preview-head.html:34]: Maybe we should have a comment here referencing the bug? Is there an open issue for it?

**Review by owensgit — COMMENTED** [review]
Thanks Paul, just one small note from me

File: .storybook/preview-head.html (hunk 1 above)
Comment (paul-request) [reply .storybook/preview-head.html:34]: Yeah I did think about that after raising the PR. I can't see an open issue for it, need t get round t raising one for it also

**Comment by paul-request** [comment]
> Thanks Paul, just one small note from me

@owensgit I managed to find some info on adding documentation-only `mdx` files, so have added those instead for the ones where there is only some docs and nothing to render to the canvas (spacing and mixins)

**Review by owensgit — APPROVED** [review]

**Pull request merged**